packages/common/src/ui/components/menus/ColorMenu/ColorMenu.test.tsx

```diff
@@ -22,7 +22,7 @@ describe('ColorMenu', () => {
     };
 
     return (
-      <TestingProvider>
+      <>
```

Comment:
Is there a reason for the move?

Reply:
There's a few like this, where `useTranslation` was used outside of the provider.. i.e.

```
const SomeComponent = () => {
    const t = useTranslation();
    return <TestingProvider> ... </...>
}
```

Can't see the `useTranslation` in the diff on this one

Reply:
ah. sneaky